### PR TITLE
TableView: Moved onClick event from "li" to anchor.

### PR DESCRIPTION
**Pull request by dlabrecq**
TableView: Moved onClick event from "li" to anchor and set first and previous buttons as disabled by default.

## Todos
- [x] cross browser test
- [ x] Are you sure it works on IE9?

## Steps to test or reproduce and link to rawgit

https://rawgit.com/dlabrecq/patternfly/PTNFLY-906-dist/dist/tests/table-view.html

### CODE REVIEW

**Comment by jgiardino** [comment]
Thanks, Dan! Looks good to me

**Comment by jgiardino** [comment]
Oh, I just noticed that the page size is set to 15 by default, but 20 results are displayed.

**Comment by dlabrecq** [comment]
Thanks. Set the default page size to be 15. Also added functionality for devs to define the initial page size.

**Comment by jgiardino** [comment]
Thanks, @dlabrecq! I see the page size working as expected now.

**Review by dtaylor113 — APPROVED** [review]

File: src/js/patternfly.dataTables.pfPagination.js [hunk 1]
```diff
@@ -126,6 +127,9 @@
     ctx._pfPagination.totalItems = $(TOTAL_ITEMS_SELECTOR, opts.paginationSelector); // Total items
     ctx._pfPagination.totalPages = $(TOTAL_PAGES_SELECTOR, opts.paginationSelector); // Total pages
 
+    // Set initial page size
+    ctx._pfPagination.currentPageSize = (opts.pageSize !== undefined) ? opts.pageSize : 15;
```
Comment (jeff-phillips-18) [inline src/js/patternfly.dataTables.pfPagination.js:131]: Is 15 a reasonable default? seem small to me.

**Review by jeff-phillips-18 — APPROVED** [review]
LGTM, one question inline.

File: src/js/patternfly.dataTables.pfPagination.js (hunk 1 above)
Comment (dlabrecq) [reply src/js/patternfly.dataTables.pfPagination.js:131]: I'll defer to @jgiardino about the 15 rows? Although, I made this configurable.

**Pull request merged**